src/covid_model_seiir_pipeline/pipeline/postprocessing/model/final_outputs.py

```diff
@@ -1,14 +1,17 @@
-from typing import Any, Callable, List
+from typing import Any, Callable, List, TYPE_CHECKING
 
 import pandas as pd
 
-from covid_model_seiir_pipeline.postprocessing.data import PostprocessingDataInterface
-from covid_model_seiir_pipeline.postprocessing.model import aggregators, loaders
+from covid_model_seiir_pipeline.pipeline.postprocessing.model import aggregators, loaders
+
+if TYPE_CHECKING:
+    from covid_model_seiir_pipeline.pipeline.postprocessing.data import PostprocessingDataInterface
```

Comment:
Is this to avoid a circular dependency? If not, why are you doing this?

Reply:
This is contract enforcement.  `model` subpackages are libraries to their respective pipeline stages and may be used anywhere.  The fact that these show up for data interfaces and specifications in lots of the model subpackages points to an incongruity in the API.  Those two object types actually need to be shared across pipeline stages.  After which I can relax this contract.

I added boilerplate comments to all the TYPE_CHECKING blocks to explain them.